### ISSUE TITLE
Could I add a function to support transactions of LTC ? If possible , what should i do

### ISSUE THREAD

**Issue by jcjcomeon** [mycelium-com/wallet-android#410]
I want to add a function to support transactions of LTC through Trezor , is it possible?
If possible , What's the general procedure?

**Comment by jcjcomeon** [comment]
I want to add a function to support transactions of LTC through Trezor , is it possible? 
If possible , What's the general procedure?

**Comment by Giszmo** [comment]
How good a programmer are you and how motivated are you? We are working on modularization and are still not finalizing the api, so if you feel like helping with the api, checkout the modularization branch. There are non-public branches for BCH and DASH but BCH will not be a full BCH support, so no integration with trezor and probably not even transaction history or the normal send screen and DASH is an abandoned branch in favor of getting it to work right with BTC as a start.

So if you don't have 200h to spare now, you probably better wait for others to put in those long hours and dumb it down for the next 1200 alt coins to be integrated.
